examples/30_extended/tasks_tutorial.py

```diff
@@ -75,15 +75,15 @@
 
 tasks = openml.tasks.list_tasks(tag='OpenML100')
 tasks = pd.DataFrame.from_dict(tasks, orient='index')
-print("First 5 of %s tasks:" % len(tasks))
+print("First 5 of {len(tasks)} tasks:")
 print(tasks.head())
 
 ############################################################################
 # Furthermore, we can list tasks based on the dataset id:
 
 tasks = openml.tasks.list_tasks(data_id=1471)
 tasks = pd.DataFrame.from_dict(tasks, orient='index')
-print("First 5 of %s tasks:" % len(tasks))
+print("First 5 of {len(tasks)} tasks:")
```

Comment:
Isn't there an `f` missing in front of the string?

Reply:
Right!